misc/deployer/build.gradle

```diff
@@ -0,0 +1,233 @@
+            port = 22
+            user = int2_user
+            password = int2_password
+            knownHosts = allowAnyHosts
+        }
+
+        stg1 {
+            role 'staging'
+            host = stg1_host
+            port = 22
+            user = stg1_user
+            password = stg1_password
+            knownHosts = allowAnyHosts
+        }
+
+        stg2 {
+            role 'staging'
+            host = stg2_host
+            port = 22
+            user = stg2_user
+            password = stg2_password
+            knownHosts = allowAnyHosts
+        }
+
+        relshare {
+            host = relshare_host
+            user = relshare_user
+            password = relshare_password
+            knownHosts = allowAnyHosts
+        }
+    }
+}
+
+project.ext.deployfolder = project.buildDir.absolutePath + '/../../../' + '/deploy'
+project.ext.allJars = []
+def version = '2.0.3'
+
+task zipJars(type: Zip, dependsOn: [getTasksByName('build', true), 'getAllJars']) {
+    outputs.upToDateWhen { false }
+    destinationDir project.buildDir
+    archiveName 'package.zip'
+    from(project.ext.allJars) {
+        exclude '*maven-plugin*'
+        exclude '*client-core*'
+        exclude '*gradle-plugin*'
+    }
+}
+
+task cleanup(type: Delete) {
+    delete project.ext.deployfolder
+}
+
+task unzipJars(type: Copy, dependsOn: ['zipJars', 'cleanup']) {
+	def path = project.buildDir.absolutePath + '/../../../' + '/zip/target/packages-'
+    def zipFile = file( path + version + '-SNAPSHOT/bundles.zip')
+    def outputDir = file(project.ext.deployfolder)
+    from zipTree(zipFile)
+    into outputDir
+}
+
+task getAllJars() << {
+    allprojects.each {
+        if (!it.extensions.extraProperties.has('allJars')) {
+            project.ext.allJars.add(it.configurations.archives.artifacts.files.files)
+        }
+
+    }
+    subprojects.each {
+        if (it.extensions.extraProperties.has('allJars')) {
+            project.ext.allJars.add(it.extensions.extraProperties.allJars)
+        }
+    }
+}
+
+getAllJars { outputs.upToDateWhen { false } }
+
+task deployDevClearCache(dependsOn: 'unzipJars', group: "Environment deployment", description: "Performs deployment on vagrant machine with karafs restart and removing cache") << {
+    ssh.run {
+        session(remotes.vagrant) {
+            execute 'sudo /etc/init.d/karaf stop', ignoreError: true
+            execute 'sudo rm -rf /opt/aet/karaf/current/data/cache/*'
+            execute 'sudo rm -rf /opt/aet/karaf/current/data/generated-bundles/*'
+            execute 'sudo rm -rf /opt/aet/karaf/current/data/tmp/*'
+            execute 'rm -rf /opt/aet/karaf/current/data/log/*'
+            execute 'sudo rm -rf /opt/aet/karaf/current/deploy/*'
+            put from: fileTree(project.ext.deployfolder), into: '/opt/aet/karaf/current/deploy'
+            put from: file(project.buildDir.absolutePath + '/../../../' + '/zip/target/aet-features.xml'), into: '/opt/aet/karaf/current/deploy'
```

Comment:
So, is it true that the OSGi configuration hasn't changed since last release?
I have an impression like without configuration deployment this is half-done.
If we are in hurry maybe at least we could create an issue for updating those Gradle scripts to deploy configuration?
What do you think?